### DISCUSSION TITLE
Extra Farmhands

### DISCUSSION THREAD

**Discussion by TheDcol** [sunflower-land/sunflower-land#3816] [Ideas]
Allow players to purchase extra Farmhands so they can equip wearables. For some players, the current design is not enough to satisfy their needs/wants. These players would be willing to spend more BBs if it meant allowing their wearables to be used.

Should we introduce a separate farmhand buying section? Add mannequins that can be placed on a farm?

**Comment by Dionis404** [comment]
While I was pondering the topic of automating oil harvesting, it occurred to me that a FarmHand should be busy working on a farm. 

For example, for a harvesting machine to work, the player should have 1 free FarmHand that can be sent to work. 

I think there is some logic to this, and the point of buying new FarmHands is to make them not just clothes dummies, but farm workers.
